app-ios/tutanota/Sources/Remote/RemoteBridge.swift

```diff
@@ -11,8 +11,8 @@ class RemoteBridge : NSObject, NativeInterface {
   var commonNativeFacade: CommonNativeFacade!
   private var mobileFacade: MobileFacade!
   private var globalDispatcher: IosGlobalDispatcher!
-  // We make the IosSqlCipherFacade available in here in order to be able to close the offline database when the app terminates
-  private var sqlCipherFacade = IosSqlCipherFacade()
+  // We make the IosSqlCipherFacade available publicly in order to be able to close the offline database when the app terminates in ViewController.swift
+  var sqlCipherFacade = IosSqlCipherFacade()
```

Comment:
didn't I suggest moving it into viewController and passing it in? I'm confused, I'm sorry

Reply:
ah I see, now I know what you mean, thanks